### DISCUSSION TITLE
Can Debezium Lose Events? - Gunnar Morling

### DISCUSSION THREAD

**Discussion by giscus** [gunnarmorling/discussions.morling.dev#311] [Announcements]
# Can Debezium Lose Events? - Gunnar Morling

This question came up on the Data Engineering sub-reddit the other day:
Can Debezium lose any events?
I.e. can there be a situation where a record in a database get inserted, updated, or deleted, but Debezium fails to capture that event from the transaction log and propagate it to downstream consumers?

https://www.morling.dev/blog/can-debezium-lose-events/

**Comment by 22maksim** [comment]
I have been setting up CDC for two weeks and I am tired of doing it. You helped me. Thank you
